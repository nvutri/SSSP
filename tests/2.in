p sp 6 20
a 1 1 10
a 1 5 -2
a 2 1 3
a 2 2 9
a 2 6 3
a 3 2 7
a 3 3 8
a 3 4 7
a 4 1 3
a 4 3 8
a 4 4 7
a 4 5 5
a 5 2 8 
a 5 4 9
a 5 5 9
a 5 6 13
a 6 2 4 
a 6 5 2
a 6 6 -1
