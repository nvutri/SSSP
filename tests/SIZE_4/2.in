c random graph n=32 m=128 lo=0 hi=16 seed=272758093
p sp 32 128
a 18 20 14
a 20 6 15
a 6 21 6
a 21 13 13
a 13 14 11
a 14 12 7
a 12 26 4
a 26 19 4
a 19 5 10
a 5 29 13
a 29 9 8
a 9 17 13
a 17 28 13
a 28 11 11
a 11 3 13
a 3 27 10
a 27 32 12
a 32 1 13
a 1 23 15
a 23 2 4
a 2 15 13
a 15 30 15
a 30 16 2
a 16 22 2
a 22 31 8
a 31 10 5
a 10 7 12
a 7 24 1
a 24 25 13
a 25 8 3
a 8 4 3
a 4 18 16
a 22 19 4
a 26 32 8
a 1 4 3
a 17 2 15
a 22 5 15
a 13 14 14
a 7 9 13
a 2 22 3
a 23 7 8
a 27 30 6
a 23 6 6
a 29 19 11
a 8 16 16
a 31 6 4
a 14 12 2
a 7 10 13
a 20 5 8
a 12 30 1
a 28 19 5
a 16 25 2
a 23 17 13
a 10 19 13
a 27 10 4
a 30 6 0
a 26 10 11
a 14 26 8
a 7 2 9
a 9 17 2
a 23 1 10
a 24 19 3
a 3 1 4
a 14 29 15
a 7 3 1
a 7 5 14
a 29 5 1
a 5 25 2
a 23 12 6
a 26 28 4
a 6 14 7
a 10 22 9
a 20 10 4
a 18 12 2
a 22 10 6
a 28 19 14
a 13 4 15
a 30 22 6
a 15 9 8
a 9 15 12
a 11 13 1
a 1 28 5
a 1 26 14
a 13 14 3
a 23 12 10
a 22 11 10
a 31 10 8
a 27 21 4
a 28 22 9
a 22 30 12
a 5 6 15
a 22 2 13
a 6 19 11
a 31 21 5
a 32 15 15
a 1 9 3
a 13 31 13
a 6 17 3
a 21 7 8
a 6 8 11
a 28 12 4
a 20 16 10
a 9 16 16
a 19 4 9
a 19 26 14
a 3 10 1
a 11 22 1
a 23 22 15
a 3 24 1
a 24 31 10
a 26 25 3
a 6 32 12
a 17 16 3
a 31 9 10
a 20 15 1
a 20 22 8
a 19 23 1
a 22 7 11
a 2 28 11
a 28 3 4
a 27 23 11
a 30 7 3
a 32 14 10
a 1 15 13
a 21 17 11
a 6 22 14
a 15 5 10
a 10 20 7
