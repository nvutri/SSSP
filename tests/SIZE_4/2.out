1 0
2 15
3 9
4 3
5 20
6 10
7 13
8 13
9 3
10 10
11 16
12 9
13 17
14 17
15 13
16 8
17 5
18 19
19 10
20 17
21 16
22 10
23 11
24 10
25 10
26 13
27 19
28 5
29 32
30 10
31 18
32 21
